runtime-version/main.go

```diff
@@ -90,8 +90,13 @@ func doQuery(cmd *cobra.Command, args []string) {
 				versionCounts[versionString] = versionCounts[versionString] + 1
 				totalNodes += 1
 
-				// todo: broken if an entity has multiple nodes running different versions
-				entityVersions[node.EntityID] = versionString
+				// Store runtime version info for current entity.
+				curEntityVersions := entityVersions[node.EntityID]
```

Comment:
Not that it really matters, but this could be simplified a bit: 

```go
if _, ok := entityVersions[node.EntityID]; !ok {
  entityVersions[node.EntityID] = make(map[string]bool)
}
entityVersions[node.EntityID][versionString] = true
```

Reply:
Thanks for the suggestion!
I've updated the code.